alpha/apps/kaltura/lib/kMrssManager.php

```diff
@@ -295,10 +295,15 @@ protected static function appendFlavorAssetMrss(flavorAsset $flavorAsset, Simple
 		$content->addAttribute('extension', $flavorAsset->getFileExt());
 		$content->addAttribute('createdAt', $flavorAsset->getCreatedAt());
 
+		// get the file size
+		$syncKey = $flavorAsset->getSyncKey(flavorAsset::FILE_SYNC_FLAVOR_ASSET_SUB_TYPE_ASSET);
+		list($fileSync, $local) = kFileSyncUtils::getReadyFileSyncForKey($syncKey, false, false);
```

Comment:
You're right, fixed.
I confused it with kFileSyncUtils::file_get_contents which actually pulls the content of the file sync